openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/PoliciesDetailPage/PoliciesDetail.less

```diff
@@ -32,10 +32,11 @@
     .ant-collapse-item {
       .ant-collapse-header {
         padding: 16px;
+        align-items: flex-start;
         .ant-collapse-arrow {
           color: @text-color;
           font-size: 14px;
-          margin-right: 6px;
+          padding: 5px 0;
```

Comment:
Do we need this anymore as we have changed the rule card from collapsible to normal?

Reply:
Updated